products/monitoring/api.yaml

```diff
@@ -0,0 +1,754 @@
+                       - :REDUCE_COUNT
+                       - :REDUCE_COUNT_TRUE
+                       - :REDUCE_COUNT_FALSE
+                       - :REDUCE_FRACTION_TRUE
+                       - :REDUCE_PERCENTILE_99
+                       - :REDUCE_PERCENTILE_95
+                       - :REDUCE_PERCENTILE_50
+                       - :REDUCE_PERCENTILE_05
+              - !ruby/object:Api::Type::NestedObject
+                name: trigger
+                description: |
+                  The number/percent of time series for which
+                  the comparison must hold in order for the
+                  condition to trigger. If unspecified, then
+                  the condition will trigger if the comparison
+                  is true for any of the time series that have
+                  been identified by filter and aggregations.
+                properties:
+                 - !ruby/object:Api::Type::Double
+                   name: percent
+                   description: |
+                     The percentage of time series that
+                     must fail the predicate for the
+                     condition to be triggered.
+                 - !ruby/object:Api::Type::Integer
+                   name: count
+                   description: |
+                     The absolute number of time series
+                     that must fail the predicate for the
+                     condition to be triggered.
+              - !ruby/object:Api::Type::String
+                name: duration
+                description: |
+                  The amount of time that a time series must
+                  fail to report new data to be considered
+                  failing. Currently, only values that are a
+                  multiple of a minute--e.g. 60s, 120s, or 300s
+                  --are supported.
+                required: true
+              - !ruby/object:Api::Type::String
+                name: filter
+                description: |
+                  A filter that identifies which time series
+                  should be compared with the threshold.The
+                  filter is similar to the one that is
+                  specified in the
+                  MetricService.ListTimeSeries request (that
+                  call is useful to verify the time series
+                  that will be retrieved / processed) and must
+                  specify the metric type and optionally may
+                  contain restrictions on resource type,
+                  resource labels, and metric labels. This
+                  field may not exceed 2048 Unicode characters
+                  in length.
+           - !ruby/object:Api::Type::String
+             name: name
+             description: |
+               The unique resource name for this condition.
+               Its syntax is: projects/[PROJECT_ID]/alertPolicies/[POLICY_ID]/con
+               ditions/[CONDITION_ID] [CONDITION_ID] is assigned
```

Comment:
this should be on the same line as the previous one so it doesn't end up with a space in between "con" and "ditions" in the generated markdown